packages/edit-site/src/components/block-editor/index.js

```diff
@@ -124,6 +124,7 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 					<BlockList
 						className="edit-site-block-editor__block-list wp-site-blocks"
 						__experimentalLayout={ LAYOUT }
+						renderAppender={ isTemplatePart ? false : undefined }
```

Comment:
Should we still render the appender if the template is empty? As stated in the original issue's title.

Reply:
If you remove all blocks, the default empty paragraph prompt appears:

<img width="1354" alt="Screenshot 2021-12-08 at 09 53 57" src="https://user-images.githubusercontent.com/846565/145187597-59bb77ac-2361-475b-b669-aa0b221ddb73.png">

This probably warrants some design exploration (it might be good to surface patterns more prominently here), and shouldn't hold up this PR.

Reply:
Yeah the paragraph appender will still appear. Agree it could use some love 🙂

Reply:
I think we can address that as a part of https://github.com/WordPress/gutenberg/issues/28737, and this probably elevates the importance of looking in to that issue for 6.0.